oemof_b3/tools/geo.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Point
+
+
+def load_regions_file(file_path):
+    """
+    loads a geopackage containing all regions of Germany and filters for geodata of Berlin and
+    all regions in Brandenburg
+
+    Outputs
+    --------------
+    geopandas.GeoDataFrame
+        with geoinformation of Berlin and all regions in Brandenburg
+    """
+    file_path = r"C:\Users\meinm\Documents\Git\oemof-B3\raw\boundaries_germany_nuts3.gpkg"
```

Comment:
Hard-coded file path, probably forgot to delete this line?

Reply:
Yes, it is removed by the next commit